src/status_im2/contexts/wallet/home/view.cljs

```diff
@@ -13,6 +13,8 @@
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
+(declare account-cards)
```

Comment:
why do we need this? 🤔

Reply:
it's a hack to fix cross-reference issue: new-account points to account-cards before the declaration of the latter — so i'm declaring it beforehand

Reply:
btw this is probably as a re-frame sub in the actual view. 
similar to how it's done in create account ->
src/status_im2/contexts/wallet/create_account/view.cljs

<img width="694" alt="Screenshot 2023-10-31 at 16 45 01" src="https://github.com/status-im/status-mobile/assets/22799766/9d9f583e-9012-49a7-b6a0-8503190776b3">

I'd say go with the approach in create account page and once we have some watch only accounts added we can update the sub to filter for watch only accounts 👍